docs/installation/cloud-ksql-migration-guide.rst

```diff
@@ -0,0 +1,111 @@
+.. _cloud-ksql-migration-guide:
+
+Migrate |ccloud| KSQL from the preview version to the production-available version
+##################################################################################
+
+To deliver the benefits of new features and improvements as soon as they become
+available, new versions of |ccloud| KSQL are deployed frequently. Most of these
+deployments are invisible to you as a user, requiring no action on your part.
+
+The production-available version of |ccloud| KSQL is not backward-compatible 
+with the preview version, but after the release of the production-available
+version, we're committed to preserving backward compatibility.

```

Comment:
@derekjn is this strictly true? I thought we had plans for supporting multiple versions in the future, in order to account for breaking changes if necessary.

Reply:
Yes, I only added this part after @rodesai reviewed the initial draft (which didn't make this strong of a commitment) and suggested this.